scripts/in-toto/keys.py

```diff
@@ -0,0 +1,171 @@
+# Imports.
+
+# 1st-party.
+import os
+import shutil
+
+# 2nd-party.
+from typing import Any, Dict, List
+
+# 3rd-party.
+from securesystemslib.interface import (
+    generate_and_write_ed25519_keypair,
+    import_ed25519_privatekey_from_file,
+    import_ed25519_publickey_from_file,
+)
+
+# Constants.
+# Where we use by default to store Notary private keys.
+KEYSTORE_DIR = os.path.expanduser('~/.signy/private')
```

Comment:
IIRC, Signy has a configurable config directory - do we want to keep the same convention?